unpick-definitions/lightmap_coordinates.unpick

```diff
@@ -0,0 +1,222 @@
+	param 4 lightmap_coordinates
+
+target_method net/minecraft/client/render/block/entity/ChestBlockEntityRenderer render (Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumer;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;FII)V
+	param 6 lightmap_coordinates
+
+target_method net/minecraft/client/render/block/entity/SkullBlockEntityRenderer renderSkull (Lnet/minecraft/util/math/Direction;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;ILnet/minecraft/client/render/block/entity/SkullBlockEntityModel;Lnet/minecraft/client/render/RenderLayer;)V
+	param 4 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/DragonFireballEntityRenderer produceVertex (Lnet/minecraft/client/render/VertexConsumer;Lnet/minecraft/util/math/Matrix4f;Lnet/minecraft/util/math/Matrix3f;IFIII)V
+	param 2 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/EnderDragonEntityRenderer renderCrystalBeam (FFFFILnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 6 lightmap_coordinates
+target_method net/minecraft/client/render/entity/EnderDragonEntityRenderer$DragonEntityModel setLimbRotation (Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumer;IIFLnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;Lnet/minecraft/client/model/ModelPart;F)V
+	param 2 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/EntityRenderDispatcher getLight (Lnet/minecraft/entity/Entity;F)I
+	return lightmap_coordinates
+target_method net/minecraft/client/render/entity/EntityRenderDispatcher render (Lnet/minecraft/entity/Entity;DDDFFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 11 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/EntityRenderer getLight (Lnet/minecraft/entity/Entity;F)I
+	return lightmap_coordinates
+target_method net/minecraft/client/render/entity/EntityRenderer renderLabelIfPresent (Lnet/minecraft/entity/Entity;Lnet/minecraft/text/Text;Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 4 lightmap_coordinates
+target_method net/minecraft/client/render/entity/EntityRenderer render (Lnet/minecraft/entity/Entity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 5 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/ExperienceOrbEntityRenderer vertex (Lnet/minecraft/client/render/VertexConsumer;Lnet/minecraft/util/math/Matrix4f;Lnet/minecraft/util/math/Matrix3f;FFIIIFFI)V
+	param 9 lightmap_coordinates
```

Comment:
```suggestion
	param 10 lightmap_coordinates
```

Reply:
The arguments numbering is quite confusing, once it starts at 0, once it starts at 1, it's not consistent at all in Enigma mappings and it leads to quite the confusion.

Reply:
In unpick format, it is the index of the argument in the argument array in the descriptor than the actual slot of the variable, I think, so you don't apply an offset for instance methods. Forgot how it treats long/double though.